src/dispatch/incident/flows.py

```diff
@@ -265,6 +271,30 @@ def incident_create_flow(*, organization_slug: str, incident_id: int, db_session
             db_session=db_session,
         )
 
+    return incident
+
+
+@background_task
+def incident_create_resources_flow(
+    *, organization_slug: str, incident_id: int, db_session=None
+) -> Incident:
+    """Creates all resources required for new incidents and initiates incident response workflow."""
```

Comment:
I think this docstring makes sense for the `incident_create_flow` function, but not this one. I'd change it to something like "Creates all resources for an existing incident" or something like that.